idunn/blocks/opening_hour.py

```diff
@@ -86,14 +66,17 @@ def from_es(cls, es_poi, lang):
             return None
 
         ohr = hoh.render_field(clean_raw, universal=False)
-        description = ohr.full_description()
-        days, is247 = from_desc(description)
 
-        status = ''
+        # plaintext_week = ohr.plaintext_week_description()
+
+        # try:
+        #    days = [{'day_name' d.split(':', 1)[0]: d.split(':', 1)[1].strip()} for d in plaintext_week.splitlines()]
```

Comment:
exception should be use only sparcingly. check `next`

(and I fhink we should clean this if not used